R/tidy-upkeep.R

```diff
@@ -111,6 +111,42 @@ if (year <= 2022) {
     )
   }
 
+  if (year <= 2023) {
+
+    desc <- proj_desc()
+
+    bullets <- c(bullets,
+     "2023",
+     "",
+     "Posit updates:",
+     "",
+     todo('Update copyright holder in DESCRIPTION: `person(given = "Posit, PBC", role = c("cph", "fnd"))`',
+          rstudio_pkg && !rstudio_person_ok),
+     todo("Double check license file uses '[package] authors' as copyright holder. Run `use_mit_license()`",
+          grepl("MIT", desc$get_field("License"))),
+     todo("Update email addresses *@rstudio.com -> *@posit.co",
+          any(grepl("rstudio", desc$get_authors()))),
+     todo("`usethis::use_tidy_coc()`"),
+     "",
+     "pkgdown:",
+     "",
+     todo("Update pkgdown site using instructions at <https://tidytemplate.tidyverse.org>"),
+     todo("Ensure pkgdown `development` is `mode: auto` in pkgdown config"),
+     todo("Submit PR [here](https://github.com/rstudio/aws-main/tree/main/zones) adding your site to the appropriate domain (eg., r-lib, tidyverse, tidymodels) and set url in GitHub Settings > Pages > Custom Domain"),
+     todo("Re-publish released site; see [How to update a released site](https://pkgdown.r-lib.org/dev/articles/how-to-update-released-site.html)"),
```

Comment:
Didn't we do all these in the last spring cleaning?

Reply:
It was on the list for last year though I understood that maybe not all of them were done, and it was suggested to me that they go in this year to finish them up. But I don't have the history so happy to change/remove!

Reply:
I'd suggested reviewing the ones for last year on the packages we hadn't 'spring cleaned' last year, since there were a few important ones to update. That could be handled in a different way though, by us saying 'check to see if you did the 2022 things', for instance.

Reply:
Yeah, we definitely have **not** done the pkgdown refresh for all of our packages at this point. But I think the checklist items in the 2022 section mostly cover pkgdown modernization already, so we could let this ride. The item about making a PR re: the subdomain is novel, though.

Reply:
I wonder if adding the subdomain would be better added to the tidytemplate instructions than in the checklist?

Reply:
Yeah, I think tidytemplate is a better home for that advice.

Reply:
PR opened at https://github.com/tidyverse/tidytemplate/pull/76